include/dfrn.php

```diff
@@ -424,11 +426,17 @@ public static function relocate($owner, $uid) {
 
 		$rp = q("SELECT `resource-id` , `scale`, type FROM `photo`
 				WHERE `profile` = 1 AND `uid` = %d ORDER BY scale;", $uid);
+
```

Comment:
This killing here is bad. These would only be missing profile pictures. So it would be enough to just don't add the xml elements.